tsconfig.json

```diff
@@ -0,0 +1,64 @@
+{
+  "compilerOptions": {
+    /* Basic Options */
+    "target": "ES2015", /* Specify ECMAScript target version: 'ES3' (default), 'ES5', 'ES2015', 'ES2016', 'ES2017','ES2018' or 'ESNEXT'. */
+    "module": "commonjs", /* Specify module code generation: 'none', 'commonjs', 'amd', 'system', 'umd', 'es2015', or 'ESNext'. */
+    // "lib": [],                             /* Specify library files to be included in the compilation. */
+    // "allowJs": true,                       /* Allow javascript files to be compiled. */
```

Comment:
should "allowJs" not be true?

Reply:
I fixed by changing config.js to config.ts.
Will try this solution also

Reply:
It works but I also need to add
```
  "include": [
    "src/**/*.ts",
    "src/**/*.js",
  ],
```

Will add it to the current config to allow JS files to be included in the library.
thanks Rajiv!